README.md

```diff
@@ -415,7 +417,9 @@ Users can add a name and optionally a public key. If `public_key` is not provide
   )
 ```
 
-#### vis variable
+#### vsi variable
+
+Note - vsi image changes are not handled through the module to avoid data loss / outage.
```

Comment:
This comment isn't clear to me. Which image are we taking about? 

However it changes, rephrase in the active voice. For example, 
You can't make changes to the VSI image with this module. That restriction is in place so that you don't inadvertently create an outage or lose data

(or something to that effect)